### ISSUE TITLE
Error when can't connect to KNX Ethernet Gateway

### ISSUE THREAD

**Issue by mishoboss** [estbeetoo/node-red-contrib-knx#1]
Hi, first - nice work! :) However I faced an issue. When the node can't connect to the KNX gateway for one reason or another (e.g. in my case the Weinzierl Ethernet gateway allows only one simultaneous connection and it was used) it gives this:

```
2015-09-08 07:32:18: new KnxControllerNode, config: {"id":"2f4a6847.d0b598","type":"knx-controller","host":"192.168.80.101","port":"3761","mode":"tunnel/unicast"}
2015-09-08 07:32:18: new KNX-IN, config: {"id":"ae26c971.51d938","type":"knx-in","name":"KNX IN","controller":"2f4a6847.d0b598","x":145,"y":229,"z":"26009041.d9ff7","wires":[["9d6e1baa.6291e8"]]}
2015-09-08 07:32:18: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:32:18: new Knx-OUT, config: {"id":"38d3b5c8.c72c4a","type":"knx-out","name":"KNX OUT","controller":"2f4a6847.d0b598","x":681,"y":438,"z":"26009041.d9ff7","wires":[]}
8 Sep 10:32:18 - [info] Started flows
2015-09-08 07:32:23: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:32:43: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:33:03: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:33:23: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:33:43: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:34:03: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:34:23: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:34:43: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
2015-09-08 07:35:03: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
(node) warning: possible EventEmitter memory leak detected. 11 connected listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at KnxConnectionTunneling.addListener (events.js:179:15)
    at KnxConnectionTunneling.once (events.js:204:8)
    at KnxConnectionTunneling.Connect (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:94:10)
    at null.<anonymous> (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:90:22)
    at Timer.listOnTimeout (timers.js:110:15)
2015-09-08 07:35:23: connecting to knxjs server at 192.168.80.101:3761 in mode[tunnel/unicast]
(node) warning: possible EventEmitter memory leak detected. 11 event listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at KnxConnectionTunneling.addListener (events.js:179:15)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/knx.js:275:29
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/knx.js:57:29
    at KnxConnectionTunneling.<anonymous> (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:84:25)
    at KnxConnectionTunneling.g (events.js:199:16)
    at KnxConnectionTunneling.emit (events.js:104:17)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:163:22
    at tryCallOne (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/lib/core.js:37:12)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/lib/core.js:103:15
    at flush (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/node_modules/asap/raw.js:50:29)
(node) warning: possible EventEmitter memory leak detected. 11 status listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at KnxConnectionTunneling.addListener (events.js:179:15)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/knx.js:276:29
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/knx.js:57:29
    at KnxConnectionTunneling.<anonymous> (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:84:25)
    at KnxConnectionTunneling.g (events.js:199:16)
    at KnxConnectionTunneling.emit (events.js:104:17)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/src/KnxConnectionTunneling.js:163:22
    at tryCallOne (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/lib/core.js:37:12)
    at /Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/lib/core.js:103:15
    at flush (/Users/Misho/.node-red/node_modules/node-red-contrib-knx/node_modules/knx.js/node_modules/promise/node_modules/asap/raw.js:50:29)

```

**Comment by aborovsky** [comment]
Thank you for response, mishoboss!
Fixed in version 0.3.1 (will be npm published today).

But, this is not a reason of unable to connect to your Weinzierl Ethernet gateway.
This is a consequence.

May I ask you to check udp port (3761) of your gateway being  available?
Howto check listening udp port: http://adityo.blog.binusian.org/?p=787

By the way: what is your gateway's model?

**Comment by aborovsky** [comment]
v0.3.1 published!
